ftl/core/decks.ftl

```diff
@@ -2,6 +2,7 @@ decks-add-new-deck-ctrlandn = Add New Deck (Ctrl+N)
 decks-are-you-sure-you-wish-to = Are you sure you wish to delete { $val }?
 decks-build = Build
 decks-cards-selected-by = cards selected by
+decks-confirm-deletion = Are you sure you want to delete all selected decks including { $count } cards?
 decks-create-deck = Create Deck
```

Comment:
Wording suggestion:

... all selected decks, and the { $count →
  [one] { $count } card
  *[other] { $count } cards
} they contain?

Note we also need to list out the plurals in this and the other strings, as otherwise we'll get "1 cards" in English, and it will make life harder for the translators. "1 card" would be clearer than "{ $count } card", but that confuses some translators when languages have different plural rules.

Reply:
Thanks, I missed that. If you agree above, I will just delete these prompt strings and replace them with info strings as mentioned.

Reply:
Sounds good